build/mirror-repo.yml

```diff
@@ -0,0 +1,19 @@
+name: Mirror-to-client-repo
+
+on:
+  pull_request:
+    branches:
+      - development
+
+jobs:
+  to_ritestream:
```

Comment:
Since this is a template I don't think we should leave the name of a specific project may just call this to_mirror